apps/movie-quotes-api/.env.example

```diff
@@ -1,5 +1,5 @@
 PORT=3042
 PLT_SERVER_HOSTNAME=127.0.0.1
 PLT_SERVER_LOGGER_LEVEL=info
-PLT_SERVER_CORS_ORIGIN=http://localhost:3000
+PLT_SERVER_CORS_ORIGIN=http://127.0.0.1:3000
```

Comment:
This is the usual problem. I think we should use `127.0.0.1` because it will work also on linux.